services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -516,7 +516,7 @@ public VeniceHelixAdmin(
       throw new VeniceException("Failed to connect to ZK within " + ZkClient.DEFAULT_CONNECTION_TIMEOUT + " ms!");
     }
     this.admin = new ZKHelixAdmin(zkClientForHelixAdmin);
-    this.helixAdminClient = new ZkHelixAdminClient(multiClusterConfigs, metricsRepository);
+    this.helixAdminClient = new ZkHelixAdminClient(commonConfig, multiClusterConfigs, metricsRepository);
```

Comment:
Can we make `ZkHelixAdminClient` extract common config from `multiClusterConfigs.getCommonConfig()` instead?